### ISSUE TITLE
Cannot integrate with PMC

### ISSUE THREAD

**Issue by vfxturjo** [dumbeau/AutoHotPie#31]
Hi, 
this is PMC -> https://www.macrocreator.com/
it is a macro creator for AutoHotKey.

I wanted to make a slice in AutoHotPie that sends f13 as a sendkey, and in PMC, i added f13 as an input for a ahk script.

i tried different keys and key combinations but for some reason PMC doesnt get shortcut from AHP.
when i normally press my keyboard, PMC works (gets the keys)
but when i send keys from AHP, it doesnt get the keys

am i doing something wrong?

![image](https://user-images.githubusercontent.com/44445014/157404553-7aa94670-7f81-4052-9b0a-5276b044fa00.png)

**Comment by dumbeau** [comment]
I don't know what PMC is. Could you elaborate?

**Comment by vfxturjo** [comment]
> I don't know what PMC is. Could you elaborate?

sure.
PMC = Pulover's macro creator

it can record user actions and writes AHK code
it has a GUI for creating different AHK tasks too.

example:
![image](https://user-images.githubusercontent.com/44445014/157405158-70844ab5-ce32-4834-8956-a8a646fe422a.png)

**Comment by vfxturjo** [comment]
I'm trying to integrate both because:
with AHP I can use mouse gestures to _trigger_ things easily, and in a categorized way (menus and submenus)
with PMC I can make my custom AHK scripts easily, _modify_ them, iterate on improving on them.

I don't know if the issue is with PMC or AHP. But I want to know
combining both world can be very useful I think

**Comment by dumbeau** [comment]
Ohhh I should've known PMC... of course!

So, the issue you're running into is that one AHK script cannot intercept a simulated keystroke from another AHK script.  What you'll have to do is export your macro from PMC as a standalone ahk script with no hotkey and use the 'Run File' function in AutoHotPie to launch the script.  Lmk if you need me to more specific on this.

What nice about that is that you also won't have to sacrifice a garbage key this way either.

**Comment by vfxturjo** [comment]
Thank you for your solution!

BTW, I tried adding hotkey in a AHKey script, then keep it running in the background, then use AHPie to trigger the code.

like this... this AHKey was running:
```
F13::
Copy_link_to_this_line:
Click, Left, 1
Sleep, 10
Send, {LControl Down}{3}{a}{b}{k}
Sleep, 30
Send, {v}{LControl Up}{Enter}
Return
```

then i sent F13 using AHPie. it worked 😄 

i know the reason...
![image](https://user-images.githubusercontent.com/44445014/157412942-86e53900-2ce4-4d42-8a8b-95eaebe1ea63.png)

when i turn it on, then another AHKey script doesnt get any keys from AHPie because it is also an AHKey script!

Anyway, I know you already know all of this. I wrote them because I am enjoying tinkering with AHPie. I love it ❤️
and I will use your solution: running AHK scripts with no hotkey, using "Run File" option.

keep it up and stay well man!
